src/store/reducers/operations.ts

```diff
@@ -0,0 +1,52 @@
+import { Reducer } from "redux";
+
+import { UserId, MemberUsername, OperationId } from "../../identifiers";
+import { OperationsAction, OperationsActionType } from "../actions/operations";
+
+export enum OperationType {
+  REQUEST_INVITE = "REQUEST_INVITE",
+  TRUST = "TRUST"
+}
+export interface RequestInvitePayload {
+  full_name: string;
+  to_uid: UserId;
+  to_mid: MemberUsername;
+}
+export interface TrustPayload {
+  to_uid: UserId;
+  to_mid: MemberUsername;
+}
+
+export interface OperationBase {
+  id: OperationId;
+  creator_mid: MemberUsername;
+  creator_uid: UserId;
+}
+
+export type Operation = OperationBase &
+  (
+    | {
+        op_code: OperationType.REQUEST_INVITE;
+        data: RequestInvitePayload;
+      }
+    | {
+        op_code: OperationType.TRUST;
```

Comment:
You don't want to include GIVE / MINT operations yet?

Reply:
Yeah I copied it before you made those I can copy those in before I merge